src/jdk.jartool/share/classes/sun/tools/jar/resources/jar.properties

```diff
@@ -235,6 +235,10 @@ main.help.opt.main.extract=\
 \  -x, --extract              Extract named (or all) files from the archive
 main.help.opt.main.describe-module=\
 \  -d, --describe-module      Print the module descriptor, or automatic module name
+main.help.opt.main.validate=\
+\      --validate             Validate a jar. This process will validate that the API\n\
+\                             exported by a multi-release jar is consistent across all\n\
+\                             different release versions.
```

Comment:
Thanks for the update. I see that that "jar archive" is used in some of the existing options so maybe tweak it to:

Validate the contents of the jar archive.
This option will validate that the API exported by ...

Reply:
Done.

FWIW, I also thought about using "jar file", but the jar tool can also take input from stdin, so that didn't seem appropriate.

Reply:
Yeah, there are inconsistencies. In other contexts we say "JAR file". In the usage output it has historically used "jar archive".